### DISCUSSION TITLE
How to compare CryoDRGN with CryoSPARC

### DISCUSSION THREAD

**Discussion by qhz991029** [zhonge/cryodrgn#361] [General]
Thank you very much for your answer and help in the previous issue.

But in the Nature Methods version paper it mentions that poses from CryoSPARC are used in the CryoDRGN training on EMPIAR-10028

> We trained the volume-decoder network with no latent variable input, using image poses obtained from C1 homogeneous refinements in cryoSPARC

I wonder how should I compare my CryoDRGN reconstruction against the CryoSPARC one.

I appreciate any feedback.

Best regards.

**Comment by michal-g** [comment]
Moving this to Discussions as replicating published results does not fall under the purview of issues with the code base.

**Comment by zhonge** [comment]
If you're interested in comparing the volume representation between cryoSPARC and cryoDRGN, I would run `cryodrgn train_nn` on the poses from the cryoSPARC reconstruction. In that case, you wouldn't need to align the two volumes before computing FSC curves. 

Note that CryoDRGN and cryoSPARC both do several different types of reconstruction, so it depends on what you want to compare.
